src/index.ts

```diff
@@ -1,33 +1,19 @@
-/* eslint-disable import/first */
 // this import must be called before the first import of tsyring
 import 'reflect-metadata';
-import { Tracing } from '@map-colonies/telemetry';
-import { createTerminus } from '@godaddy/terminus';
```

Comment:
why did you remove terminus?

Reply:
terminus has no compatibility with fastify, we'll need to use [another package](https://www.fastify.io/ecosystem/) or implement on our own.
at the moment I've implemented a healthcheck plugin, I'm not sure it accomplishes everything we'd like to.

Reply:
my bad. terminus has compatibility with fastify, added terminus instead of a custom plugin.